org_fedora_oscap/gui/spokes/oscap.py

```diff
@@ -650,26 +652,35 @@ def _update_message_store(self, report_only=False):
 
     def _resolve_rootpw_issues(self, messages, report_only):
         """Mitigate root password issues (which are not fatal in GUI)"""
-        fatal_rootpw_msgs = [msg for msg in messages
-                             if msg.origin == rule_handling.PasswdRules and msg.type == common.MESSAGE_TYPE_FATAL]
+        fatal_rootpw_msgs = [
+            msg for msg in messages
+            if msg.origin == rule_handling.PasswdRules and msg.type == common.MESSAGE_TYPE_FATAL]
+
         if fatal_rootpw_msgs:
             for msg in fatal_rootpw_msgs:
                 # cannot just change the message type because it is a namedtuple
                 messages.remove(msg)
-                messages.append(common.RuleMessage(self.__class__,
-                                                   common.MESSAGE_TYPE_WARNING,
-                                                   msg.text))
+
+                msg = common.RuleMessage(
+                    self.__class__, common.MESSAGE_TYPE_WARNING, msg.text)
+                messages.append(msg)
+
             if not report_only:
-                self.__old_root_pw = self.data.rootpw.password
+                users_proxy = USERS.get_proxy()
+
+                self.__old_root_pw = users_proxy.RootPassword
                 self.data.rootpw.password = None
-                self.__old_root_pw_seen = self.data.rootpw.seen
+                self.__old_root_pw_seen = users_proxy.IsRootpwKickstarted
                 self.data.rootpw.seen = False
```

Comment:
I don't think this fix is working correctly. You should call `users_proxy.SetRootPassword` and `users_proxy.SetRootpwKickstarted` as you do bellow. The `rootpw` is attribute is available, but useless in RHEL 8. It is no longer available in RHEL 9.